templates/includes/sidebar.html

```diff
@@ -1,4 +1,10 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+  <li><a href="{% url 'taxi:driver-detail' pk=user.id %}">{{ user.username }}</a></li><br>
+  <li><a href="{% url 'logout' %}">Logout</a></li>
```

Comment:
The `logout` URL should be correctly configured in your URL patterns. Ensure that the `name` attribute for the logout path is set to 'logout'.